docs/1-terms/F/term-filter.adoc

```diff
@@ -3,9 +3,9 @@
 // tag::EN[]
 ==== Filter
 
-Part of the pipe-and-filter architectural style that creates or transforms data.
+Part of the pipes and filters architectural style that creates or transforms data.
```

Comment:
Shouldn't "style" be "pattern" for consistency?

Reply:
done